### PR TITLE
Update StateMachine.c

### PR DESCRIPTION
**Pull request by bergdahl**
Changed wording to use "max" instead

### CODE REVIEW

**Review by dhalbert — COMMENTED** [review]
@bergdahl Are you requesting this change to make both it and your translation shorter? I have been advocating use comparison operators like `In-buffer elements must be <= 4 bytes long` for brevity, but I don't know how easily that translates.

**Comment by bergdahl** [comment]
I don't see a problem expressing the error the same way you would express them in the language itself
(Scott merged this pull request together with the fix of his strings in the last live stream).

**Comment by tannewt** [comment]
I didn't merge this during my stream. I only ran `make translate`. (See: https://github.com/adafruit/circuitpython/blob/main/locale/circuitpython.pot#L1089 is the original) @dhalbert do you want to update this PR with the rewording?

File: ports/raspberrypi/bindings/rp2pio/StateMachine.c [hunk 1]
```diff
@@ -468,12 +468,12 @@ STATIC mp_obj_t rp2pio_statemachine_write_readinto(size_t n_args, const mp_obj_t
...
     int in_stride_in_bytes = mp_binary_get_size('@', buf_in_info.typecode, NULL);
     if (in_stride_in_bytes > 4) {
-        mp_raise_ValueError(translate("In buffer elements must be 4 bytes long or less"));
+        mp_raise_ValueError(translate("In buffer elements must be max 4 bytes long"));
     }
 
     int out_stride_in_bytes = mp_binary_get_size('@', buf_out_info.typecode, NULL);
     if (out_stride_in_bytes > 4) {
-        mp_raise_ValueError(translate("Out buffer elements must be 4 bytes long or less"));
+        mp_raise_ValueError(translate("Out buffer elements must be max 4 bytes long"));
```
Comment (dhalbert) [inline ports/raspberrypi/bindings/rp2pio/StateMachine.c:476]:
```suggestion
        mp_raise_ValueError(translate("Out-buffer elements must be <= 4 bytes long"));
```

**Review by dhalbert — CHANGES_REQUESTED** [review]
OK, I will make these changes in the web interface, and will then push a `make translate`.

**Review by dhalbert — APPROVED** [review]

**Pull request merged**